features/cellular/framework/AT/ATHandler.h

```diff
@@ -24,9 +24,9 @@
 #include "PlatformMutex.h"
 #include "nsapi_types.h"
 
-#include "PlatformMutex.h"
 #include "Callback.h"
-#include <cstdarg>
+
+#include <stdarg.h>
```

Comment:
I'd be inclined to fix this the other way and turn `va_list` into `std:;va_list`, unless `<cstdarg>` itself is broken.

Reply:
Fair enough, I was a bit too scared to touch the API. Changed back the ```<cstdarg>``` and converted ```va_list``` to ```std::va_list```.
Anyway, what itches my nerves is that the compilation fails only on debug profile. Is someone leaking a ```using namespace std;``` from a ```#if _DEBUG``` block or what is going on there?